src/main/java/com/courtcanva/ccfranchise/dtos/FranchiseeDto.java

```diff
@@ -0,0 +1,22 @@
+package com.courtcanva.ccfranchise.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class FranchiseeDto {
+
+    private String businessName;
```

Comment:
as a business, in Australia, we need legal entity name and business name

Reply:
add new entityname